src/components/atoms/Text/Text.tsx

```diff
@@ -53,20 +48,20 @@ const Text = styled.span<ITextProps>`
   letter-spacing: 0;
   color: ${({ color = colors.greyDarkest }) => color};
 
-  ${({ mb = false }) =>
-    mb &&
-    `
-    display: block;
-    margin-bottom:   24px
-  `};
-
   font-size: ${({ size = 'body', capitalize }) => typography.sizes.text[capitalize ? 'small' : size]};
   line-height: ${({ size = 'body' }) => lineHeightMap[size]};
   font-weight: ${({ weight = 'regular', capitalize }) => typography.weights[capitalize ? 'bold' : weight]};
 
   font-family: ${typography.primary};
   text-align: ${({ align = 'inherit' }) => align};
   text-transform: ${({ capitalize }) => capitalize && 'uppercase'};
+
+  ${({ className = '', as }) =>
+    (!as || as === 'span') &&
+    className.split(' ').some(clss => clss.includes('mb')) &&
```

Comment:
We probably are gonna need to handle more cases than just `mb`, lets catch-up later 👍🏻

Reply:
@lluia 

How do you feel about adding some regex to detect if a spacing class is being used for example: 

```js
/[mp](\:[mp])?[tblr]?-\d+/g
```